extensions/wikia/Wall/js/MessageTopic.js

```diff
@@ -48,21 +48,21 @@
 				}
 			}
 		},
-		
+
 		handleRemove: function(e) {
 			var topic = $(e.target).closest('.topic');
 			this.removeSelection(topic.data('article-title'));
 			topic.remove();
 		},
-		
+
 		getTopics: function() {
 			var topics = [];
 			for(var selection in this.selectedEntries) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Restrict what this loop acts on by testing each property. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AForIn)